readme.txt

```diff
@@ -235,7 +235,9 @@ The plugin is produced by [Modern Tribe Inc](http://m.tri.be/18uc).
 
 = [4.5.3] TBD =
 
-* ...
+* Fix - Improved get_ticket_counts() to account for tickets with global stock enabled  [82684]
+* Fix - Improved tribe_events_count_available_tickets() to account for tickets with global stock enabled (thanks to Florian for reporting this) [81967]
+* Tweak - Optimized message displayed when errors are returned from Eventbrite API (for clarity) [80861]
```

Comment:
We should strip this one out and move to the Eventbrite changelog (but keep the first two - #82684 and #81976)